go/src/meguca/server/admin.go

```diff
@@ -40,6 +40,7 @@ var (
 	errBoardNameTaken   = common.ErrInvalidInput("board name taken")
 	errNoReason         = common.ErrInvalidInput("no reason provided")
 	errNoDuration       = common.ErrInvalidInput("no ban duration provided")
+	errMalformedRequest = common.ErrInvalidInput("malformed client request")
```

Comment:
Please provide a more specific error message. This will aid debugging in the future.

Reply:
Any ideas on what kind of specific information this needs?

Reply:
Something descriptive. I'm not sure this message will be still needed after you rework the db side though. Best start there.

Reply:
Yeah, I'm not using `errMalformedRequest` since I reverted to the old router line.